software/index.html

```diff
@@ -5,23 +5,23 @@
 category: software
 ---
 
-<section id="hetio">
+<section id="hetnetpy">
   <div class="section">
-    <h3><code>hetio</code></h3>
+    <h3><code>hetnetpy</code></h3>
     <div class="row">
       <div class="col_text">
         <p>
-          <i>Hetio</i> is a Python 3 package for creating, querying, and
+          <i>Hetnetpy</i> is a Python 3 package for creating, querying, and
           operating on hetnets. This software provides convenient data
           structures for hetnets, as well as algorithms for
           <a href="/hnep">edge prediction</a>. It is specifically tailored and
           streamlined for hetnets compared to other more generic network
           software.
```

Comment:
We should probably add somewhere in this description a note along the lines of "(renamed from `hetio` in 2019)"... since there are some uses of the package name `hetio` that we can't change and some users are familiar with this name.

Reply:
How's this:
![image](https://user-images.githubusercontent.com/8326331/61299360-e7b39e80-a7ad-11e9-9f8a-1e78645317b4.png)

Link just goes to https://github.com/hetio/hetnetpy/issues/40

Reply:
@dhimmel